src/crunker.ts

```diff
@@ -8,6 +8,7 @@ export interface CrunkerConstructorOptions {
 }
 
 export type CrunkerInputTypes = string | File | Blob;
+type MIME = 'audio/mp3' | 'audio/wav' | 'audio/ogg';
```

Comment:
I don't agree with adding this.

MIME has no impact on the actual file contents -- choosing `audio/mp3` just embeds the wav file within a file with a `.mp3` extension.

Reply:
Makes sense - I wonder if we then shouldn't state in the docs (`readme.md`) that it can be **for example audio/mp3, audio/wav, audio/ogg but the file content is in WAVE format regardless**?

Reply:
I definitely agree.

Reply:
Just updated it, that would also fix: https://github.com/jaggad/crunker/issues/73